packages/publications/src/Models/PublicationPage.php

```diff
@@ -46,6 +47,11 @@ public function compile(): string
         return $this->renderComponent();
     }
 
+    public function getType(): PublicationType
+    {
+        return $this->type;
+    }
```

Comment:
I'll cherry pick this directly to the publications branch as that can be reviewed independently of this PR; unless you want credit through a dedicated PR.

Reply:
Do what's easiest ... just cherrypick it ..

Reply:
Will do.

Reply:
Resolved in https://github.com/hydephp/develop/pull/1148